app/src/main/java/com/quran/labs/androidquran/ui/helpers/QuranDisplayHelper.java

```diff
@@ -39,6 +39,29 @@ public static Response getQuranPage(OkHttpClient okHttpClient,
     return response;
   }
 
+  // same logic used in displayMarkerPopup method
```

Comment:
I think, for this comment to make sense,  `displayRub3()` needs to come below/after `displayMarkerPopup()`